crates/messaging/src/implementors/azsbus.rs

```diff
@@ -0,0 +1,102 @@
+use azure_messaging_servicebus::service_bus::SubscriptionReceiver;
+use std::collections::HashMap;
+use std::sync::{Arc, Mutex};
+use tokio::runtime::Handle;
+use tokio::task::block_in_place;
+
+use anyhow::{Context, Result};
+use azure_messaging_servicebus::prelude::TopicClient;
```

Comment:
I wonder what's the reason to use TopicClient instead of a QueueClient.

Reply:
This is to allow subscriptions~ In the future, once we add support for a different channel type (i.e., a queue over a topic) like suggested in the `wasi-messaging` proposal, we can bring back the `QueueClient`.